src/Pos/WorkMode/Class.hs

```diff
@@ -93,6 +94,7 @@ type WorkMode ssc ctx m
       , HasLens SecurityParams ctx SecurityParams
       , HasLens TxpGlobalSettings ctx TxpGlobalSettings
       , HasLens GenesisUtxo ctx GenesisUtxo
+      , HasLens GenesisStakeholders ctx GenesisStakeholders
```

Comment:
I think it's better to have `HasLens GenesisTxpContext` instead, but it's minor.